process/elasticproc/transactions/smartContractResultsProcessor.go

```diff
@@ -152,6 +154,14 @@ func (proc *smartContractResultsProcessor) prepareSmartContractResult(
 		esdtValues = res.ESDTValues
 	}
 
+	feeInfo := &outport.FeeInfo{
+		Fee:            big.NewInt(0),
+		InitialPaidFee: big.NewInt(0),
+	}
+	if scrInfo.FeeInfo != nil {
+		feeInfo = scrInfo.FeeInfo
+	}
```

Comment:
Duplicated code here and in `process/elasticproc/transactions/transactionDBBuilder.go`
Could've extracted in a funct with some `feeInfoHandler` interface , since both scrInfo and txInfo provide getters/setters for `FeeInfo`

Reply:
refactored.